server/modules/args.py

```diff
@@ -4,6 +4,7 @@
 def get_args():
     parser = argparse.ArgumentParser()
     parser.add_argument("--config", help="yaml file", required=True)
+    parser.add_argument("--fixed_stops", help="json file with predetermined stop data")
```

Comment:
the variable will still be `args.fixed_stops`, argparse is nice to us

```suggestion
    parser.add_argument("--fixed-stops", help="json file with predetermined stop data")
```

Reply:
resolved